File: Romano/Assets/scripts/puente.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

public class puente : MonoBehaviour
{
	Animator animator;
    GameObject esparta;
    public bool puenteDerribado = false;

    private int sonando = 0;

    void Start()
	{
		animator = GetComponent<Animator>();
	}

	void Update()
	{

	}

    void OnTriggerStay(Collider other)
	{
		if (other.gameObject.tag == "espada" && esparta.GetComponent<espartano>().atacando == true)
		{
			// print("puente derribado ganaste !!! ");
			if (sonando == 0)
			{
				sonando = 1;
				GetComponent<AudioSource>().Play();
			}
			yield return new WaitForSeconds(1);
			animator.SetBool("derribado", true);
			puenteDerribado = true;
			yield return new WaitForSeconds(3);
			//carga la fase de victoria
			Application.LoadLevel("victoria");

		}
		else
		{
			animator.SetBool("derribado", false);
			puenteDerribado = false;
		}
	}
}

File: Romano/Assets/scripts/puente.js
#pragma strict

var animator:Animator;
var esparta:GameObject;
static var puenteDerribado:boolean = false;

private var sonando:int = 0;

function Start () 
{
	animator = GetComponent("Animator");
}

function Update () 
{
	
}

function OnTriggerStay(other:Collider)
{
	if(other.gameObject.tag == "espada" && esparta.GetComponent(espartano).atacando == true)
	{
		// print("puente derribado ganaste !!! ");
		if(sonando == 0)
		{
			sonando = 1;
			GetComponent.<AudioSource>().Play();
		}
		yield WaitForSeconds(1);
		animator.SetBool("derribado",true);
		puenteDerribado = true;
		yield WaitForSeconds(3);
		//carga la fase de victoria
		Application.LoadLevel("victoria");
		
	}else{
		animator.SetBool("derribado",false);
		puenteDerribado = false;
	}
}